workspaces/orchestrator/plugins/orchestrator/src/components/WorkflowEditor/WorkflowEditor.tsx

```diff
@@ -243,7 +243,6 @@ const RefForwardingWorkflowEditor: ForwardRefRenderFunction<
             if (format && workflowId && format !== workflowFormat) {
               const link = viewWorkflowLink({
                 workflowId: workflowId,
-                format: workflowFormat,
```

Comment:
do we stil need `format` in the `if` condition?

Reply:
good point, please remove the format from the condition

Reply:
there's some kind of odd rerouting here...
this code handles a case where workflowFormat is different than the format in the URL
if this is the case, rerouting won't help
this whole code should remove

Reply:
if the formats don't match we need to handle it differently
maybe show an error

Reply:
Took a look where format comes from
Backend calculates it
https://github.com/redhat-developer/rhdh-plugins/blob/main/workspaces/orchestrator/plugins/orchestrator-backend/src/service/SonataFlowService.ts#L194
Now that it comes from the backend, recalculating it in the FE is a waste
This whole code should be removed, including recalculating the format and handling the mismatch